awx/ui_next/src/components/PromptDetail/PromptDetail.jsx

```diff
@@ -34,21 +48,107 @@ function hasPromptData(launchData) {
   );
 }
 
-function formatTimeout(timeout) {
-  if (typeof timeout === 'undefined' || timeout === null) {
-    return null;
-  }
-  const minutes = Math.floor(timeout / 60);
-  const seconds = timeout - Math.floor(timeout / 60) * 60;
-  return (
-    <>
-      {minutes} <Trans>min</Trans> {seconds} <Trans>sec</Trans>
-    </>
+function removeOverrides(resource, overrides) {
+  const filteredResource = Object.keys(overrides).reduce(
+    (acc, value) => {
+      let root;
+      let nested;
+
+      ({
+        [value]: acc[value],
+        summary_fields: { [value]: acc[value], ...nested },
+        ...root
+      } = acc);
+
+      const filtered = {
+        ...root,
+        summary_fields: {
+          ...nested,
+        },
+      };
+
+      return filtered;
+    },
+    { ...resource }
```

Comment:
if you initialize with `{ ...resource, summary_fields: { ...resource.summary_fields } }` then you can safely `delete acc.summary_fields[value]` within the reducer function above since it's a copy of the original object

Reply:
That is right! Good catch.